README.md

```diff
@@ -276,6 +293,46 @@ The rule configuration files in this package (`index.js`, `typescript.js`, etc)
 
 ESLint offers several [ways to disable a rule for a line or file](https://eslint.org/docs/user-guide/configuring/rules#disabling-rules). Suppressions should be as targeted as possible and should include a comment explaining the suppression.
 
+### Migrating to ESLint Flat Configuration Format
```

Comment:
I think we're quite close with this PR. My main remaining concern is validating the impact on some real-world client projects. I'd like to see proof-of-concept migrations of one or two projects (e.g. a SystemLinkShared library and an SLE app). You mention in the PR description that you had this working for one app. Can you push that branch into a draft PR? And do the basic setup in another project like SLShared? You don't have to get the build totally passing, I just want to get a sense for the changes required and make sure we're not setting ourselves up for a huge amount of follow up work.